presto-iceberg/src/main/java/com/facebook/presto/iceberg/procedure/RollbackToSnapshotProcedure.java

```diff
@@ -58,7 +58,7 @@ public Procedure get()
                 "rollback_to_snapshot",
                 ImmutableList.of(
                         new Procedure.Argument("schema", VARCHAR),
-                        new Procedure.Argument("table", VARCHAR),
```

Comment:
I don't have strong feelings, but I think it might be better if we keep the naming consistent?

e.g. just use `schema` and `table` rather than appending the `_name` suffix only for the `table` argument.

Reply:
> e.g. just use `schema` and `table` rather than appending the `_name` suffix only for the `table` argument.

I haven't inspected it very carefully, but we will meet errors during the parsing phase when using `table` as argument name.

Reply:
Since there are parsing issues when using table, if we want to keep it consistent, should we rename `schema` as well to `schema_name`?

Reply:
Currently the procedures which declare arguments `schema` and `table` do not support named arguments, so the user cases and test cases always use sequential arguments to invoke them. So it will be ok to change the argument name `table` to `table_name`.

However the procedures which declare arguments `schema` and `table_name` support named arguments, so there already exist user cases and test cases which invoke them through named arguments.

I think maybe it's better not to change the argument name which may affect user cases. What your opinion? @ZacBlanco @imjalpreet

Reply:
I am ok to leave it as-is

Reply:
I am okay as well